src/graphql/ProductFragment.graphql

```diff
@@ -7,6 +7,30 @@ fragment ProductFragment on Product {
   description
   handle
   productType
+  priceRange {
```

Comment:
I don't think we need to include this. Let's only include functionality that we think most folks will use.